samples/kubeflow/README.md

```diff
@@ -11,52 +12,91 @@ This demo uses the default stack:
 
 ## Actors and flows
 
-A _data owner_ uploads a transactions.csv file to an s3 compatible object storage (e.g., Ceph, minio, IBM Cloud Object Storage, Amazon S3, etc.). Then, the data owner registers the asset in Egeria data catalog and adds tags to the asset to mark it as finance data. Finally, the data owner registers the credentials to access this asset in Vault credentials manager.
+A _data owner_ uploads a transactions.csv file to as s3 compatible object storage (e.g., Ceph, minio, IBM Cloud Object Storage, Amazon S3, etc.). Then, the data owner registers the asset in Egeria data catalog and adds tags to the asset to mark it as finance data. Finally, the data owner registers the credentials to access this asset in Vault credentials manager.
 
 A _data steward_ creates policies in OPA policy manager. These policies set transformations to perform on data columns when accessed for the purpose of fraud-detection.
 
 A _data user_ creates a notebook server in Kubeflow and a notebook with the business logic for creating a fraud detection model. The data user also creates a `M4DApplication` resource that references the transactions.csv asset from the data catalog. This allows the code in the notebook to load the data (after policies are applied to it).
 
 ## Installation
 
-Run the following script to install The Mesh for Data and all dependencies required for this demo. It will take a while for all containers to be ready due to image downloads.
+Run the following script to install The Mesh for Data core components and third party dependencies required for this demo.
+It will take a while for all containers to be ready due to image downloads. 
 
+```bash
+kubectl config set-context --current --namespace=m4d-system
+
+# For running on OpenShift
+WITHOUT_OPENSHIFT=false ./hack/install.sh
 
+# For running on Kind
+./hack/install.sh
+```
+
+Install kubeflow on your cluster. This sample (and code) was tasted against kubeflow v1.0.2.
+On a Kind cluster for example you can use the following script to install it:
 ```bash
-kind create cluster
-./install/install.sh
+cd samples/kubeflow/install/kubeflow
+./install_kubeflow.sh
+
+cd -
+```
+
+Install the Arrow-Flight module
+```
+kc apply -f https://raw.githubusercontent.com/IBM/the-mesh-for-data-flight-module/master/module.yaml
 ```
 
 ## Data owner instructions
 
-We already have transactions.csv stored in COS (TODO: don't assume that).
-Register the asset using the following command:
+Upload the file `data.csv` to an object-storage of your choice. For example, we tested the code against IBM's COS.
+`data.csv` contains the first 100 rows from the following [data set](https://www.kaggle.com/ntnu-testimon/paysim1/data)
```

Comment:
please check that we are distributing the data.csv file under the same license as the original data.
Please check the license at https://creativecommons.org/licenses/by-sa/4.0/

Reply:
I see now that according to the license, we must redistribute this under the same license and credit the authors. The license is `CC BY-SA 4.0`. I'm not sure what should I write and where in order to do that.

How about:
```
`data.csv` contains the first 100 rows from the following data-set [data set](https://www.kaggle.com/ntnu-testimon/paysim1/data) created by NTNU, and is shared under the ***CC BY-SA 4.0***.
```

Reply:
@tomersolomon1 maybe add a file "data.csv.license.md" ? with the license information
Do include the statement you wrote as well.

Reply:
> @roee88 Thanks, I read it now ;)
> Could you please be more specific?
> I see two main issues:
> 
> * I see can change future tense to present tense.
> * change some of the Vault-related instructions, for example write `Create secret` instead of **Create secret**.
> 
> Anything else you noticed?

Avoid using "we"

Reply:
Ok, added that line and a file containing that license, as I found in here:
https://creativecommons.org/licenses/by-sa/4.0/legalcode.txt